.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+# Description
+
+Please include a brief summary of the change.
+
+# Type of change
+
+Please put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Draft PR (breaking/non-breaking change which needs more work for having a proper functionality _[Mark this PR as ready to review only when completely ready]_)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+# Fixes/Resolves (Optional)
+
+Please list any issues that are fixed by this PR in the format `#<issue number>`. If it fixes multiple issues, please put each issue in a separate line. If this Pull Request does not fix any issues, please delete this section.
+
+# Examples/Screenshots (Optional)
+
+Please attach a screenshot or screen recording of features/fixes you've implemented to verify your changes. Please also note any relevant details for your configuration. If your changes do not affect the UI, please delete this section.
+
+Use this table template to show examples of your changes:
+
+| Before       | After        |
+| ------------ | ------------ |
+| Content Cell | Content Cell |
+| Content Cell | Content Cell |
+
+# Checklist:
+
+- [ ] I have performed a self-review of my own code
+- [ ] My changes generate no new warnings
+- [ ] Any dependent changes have been merged and published in downstream modules
+
```

Comment:
More on @Cali93 's suggestion, any change in `w3iProxy`'s api needs to be communicated. Also, we really do need to document `w3iProxy`, I'll add that as an issue.